files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/svelte_components/index.md

```diff
@@ -0,0 +1,560 @@
+---
+title: 将我们的 Svelte 应用组件化
+slug: Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_components
+---
```

Comment:
Let's also add the [`sourceCommit`](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE)

```suggestion
l10n:
  sourceCommit: e9be22eaa7416206e3f263a058f0d509a7f81f88
---
```